portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,33 @@
+package com.google.sps.data;
+
+import java.util.Date;
+
+public class Comment {
+    public Comment(int id, String name, String description, Date postDate) {
+        this.id = id;
+        this.name = name;
+        this.description = description;
+        this.postDate = postDate;
+    }
+
+    private final int id;
+    private final String name;
+    private final String description;
+    private final Date postDate;
+
+    public int getId() {
+        return id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public Date getPostDate() {
+        return postDate;
+    }
+} 
```

Comment:
Add empty line at end

Reply:
Sorry for the same mistakes, but I created some of the code before seeing the comments on the last PR. I will try my best to adhere to them from now on!